net-analyzer/netperf/netperf-2.7.0-r6.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=8
 
-inherit flag-o-matic
+inherit autotools flag-o-matic
```

Comment:
This ebuild doesn't use eautoreconf or autotools. It's not the right fix. Let's discuss it on the bug, there's far more to this.